packages/react-components/src/components/VideoTile.tsx

```diff
@@ -221,6 +229,16 @@ const VideoTileMoreOptionsButton = (props: {
   );
 };
 
+/* @conditional-compile-remove(reaction) */
+const styleSheet = document.styleSheets[0];
+/* @conditional-compile-remove(reaction) */
+const keyframes1 = `@keyframes play {
```

Comment:
name `play` could easily conflict with existing keyframe names. Consider using an auto-generated guid/uuid

Reply:
play is already a keyword for css style animation rendering. We do not need this keyframe altogether as we can control the height and rendering with in-line styles. Removing this and testing to change.